rotkehlchen/tests/unit/test_tokens.py

```diff
@@ -77,14 +77,14 @@ def test_detected_tokens_cache(ethtokens, inquirer):  # pylint: disable=unused-a
             ethtokens.detect_tokens(write_cursor, True, [addr1, addr2])
         result2, _ = ethtokens.query_tokens_for_addresses([addr1, addr2])
         call_count = etherscan_mock.call_count
-        assert call_count == initial_call_count + 2
+        assert call_count == initial_call_count + 1
```

Comment:
here is 1 call because you are calling two addresses at once right?

Reply:
Yep!